src/services/SearchService.ts

```diff
@@ -67,6 +80,17 @@ async function search<T extends SearchResponseElement>(entity: RawSearchRequestP
   }
 }
 
+async function searchValues<T extends SearchValuesResponseElement>(
+  entity: RawSearchValuesRequestPayload
+): Promise<T | null> {
+  try {
+    const response: SearchValuesResponsePayload = (await httpSearchValues.post({ entity })).data;
```

Comment:
Can data be undefined or null here?

Reply:
I believe the HttpService handles errors and returns corresponding state.
You could remove the try/catch and check the status of response.
```
  const response: SearchValueRes..... post({...});
  if (response.requestSucceeded) {
    return response.data?.results as T | null;
  } else {
    return null;
  }
```